cocotb/decorators.py

```diff
@@ -600,5 +290,26 @@ async def f(*args, **kwargs):
 
     def __call__(self, *args, **kwargs):
         inst = self._func(*args, **kwargs)
-        coro = RunningTest(inst, self)
+        coro = _RunningTest(inst, self)
         return coro
+
+
+def __getattr__(attr: str) -> typing.Any:
```

Comment:
For 3.6, you still need the `Task = _Task` etc to make those importable for users.